src/Microsoft.DotNet.Interactive/CompositeKernel.cs

```diff
@@ -213,6 +213,22 @@ protected override void SetHandlingKernel(KernelCommand command, KernelInvocatio
                 throw new NoSuitableKernelException(command);
             }
 
+            switch (command)
+            {
+                case Cancel _:
+
+                    CancelInflightCommands();
+                    ClearPendingCommands();
+                    kernel.ClearPendingCommands();
+                    break;
+                case Quit _:
+                    CancelInflightCommands();
+                    ClearPendingCommands();
+                    kernel.CancelInflightCommands();
+                    kernel.ClearPendingCommands();
+                    break;
```

Comment:
Do these two cases need to vary in terms of behavior?

Reply:
might not be the case